locals.tf

```diff
@@ -40,14 +36,14 @@ nginx:
         key: sandbox.gke.io/runtime
         operator: Equal
         value: gvisor
-
 astronomer:
   houston:
     config:
-      commander:
-        enabled: false
-      deployments:
-        logHelmValues: true
+    %{if var.smtp_uri != ""}
+        email:
+          enabled: true
+          smtpUrl: var.smtp_uri
```

Comment:
since we are in a string block, you've got to wrap this in ${}
```
          smtpUrl: ${var.smtp_uri}
```
see 'baseDomain' above